src/app/components/arlas-wui-root/arlas-wui-root.component.html

```diff
@@ -80,6 +80,9 @@
                 </div>
               </div>
             </div>
+            <div class="arlas-rasters-manager" [class.arlas-rasters-manager_with_geocoding]="enableGeocodingFeature" *ngIf="visualizeService.isRasterOnMap">
```

Comment:
For the class name, to respect BEM, it could be arlas-rasters-manager--geocoding

Reply:
fixed